tensorflow_probability/python/math/special_test.py

```diff
@@ -194,6 +194,37 @@ def testBetaincBroadcast(self):
     x = np.ones([7, 1, 1, 2], dtype=np.float32)
     self.assertAllEqual([7, 5, 3, 2], tfp_math.betainc(a, b, x).shape)
 
+  def testBetaincHalfPrecision(self):
+    a = tf.constant([0.4, 0.4, 0.4, 0.4, -1., 0.4, 0.4], dtype=tf.float16)
+    b = tf.constant([0.6, 0.6, 0.6, 0.6, 0.6, -1., 0.6], dtype=tf.float16)
+    x = tf.constant([0.0, 0.1, 0.9, 1.0, 0.5, 0.5, -1.], dtype=tf.float16)
+    result = tfp_math.betainc(a, b, x)
+
+    self.assertEqual(a.dtype, result.dtype)
+
+    expected_result = tfp_math.betainc(
+        *[tf.cast(z, tf.float32) for z in [a, b, x]])
+    expected_result = tf.cast(expected_result, a.dtype)
+
+    self.assertAllEqual(*self.evaluate([expected_result, result]))
+
+  @test_util.disable_test_for_backend(
+      disable_numpy=True, disable_jax=False,
+      reason="Numpy does not support brain floating point.")
+  def testBetaincBrain(self):
```

Comment:
`testBetaincBFloat16`, and probably change the reason to `bfloat16`

Reply:
Done (here and elsewhere).